CMakeLists.txt

```diff
@@ -214,6 +233,8 @@ if(BLAS_VENDOR STREQUAL "Intel" OR BLAS_VENDOR STREQUAL "MKL")
   set(MKLROOT $ENV{MKLROOT})
   message(STATUS "Attempting to use Intel's BLAS/LAPACK (MKL)")
   if(CMAKE_Fortran_COMPILER_ID STREQUAL "Intel")
+    message(STATUS "Look for MKL in ${MKLROOT}")
+    set(MKLDIR "${MKLROOT}/lib/intel64")
```

Comment:
I don't see where you are using `MKLDIR`. If you are not using it you might as well remove this line.

Reply:
Will remove it